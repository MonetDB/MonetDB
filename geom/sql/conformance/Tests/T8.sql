-- Conformance Item T8
SELECT ST_AsText(boundary) FROM named_places WHERE name = 'Goose Island';
