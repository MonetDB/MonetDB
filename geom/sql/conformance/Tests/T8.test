query T rowsort
SELECT ST_AsText(boundary) FROM named_places WHERE name = 'Goose Island'
----
POLYGON ((67 13, 67 18, 59 18, 59 13, 67 13))

