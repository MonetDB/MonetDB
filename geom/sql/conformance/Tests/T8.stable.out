stdout of test 'T8` in directory 'geom/sql/conformance` itself:


# 14:02:52 >  
# 14:02:52 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=35167" "--set" "mapi_usock=/var/tmp/mtest-14721/.s.monetdb.35167" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/alvanaki/INSTALL/MonetDB-public/var/MonetDB/mTests_geom_sql_Tests_conformance" "--set" "mal_listing=0"
# 14:02:52 >  

# MonetDB 5 server v11.20.0
# This is an unreleased version
# Serving database 'mTests_geom_sql_Tests_conformance', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs and 128bit integers dynamically linked
# Found 15.356 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2014 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://sibuyan.da.cwi.nl:35167/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-14721/.s.monetdb.35167
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

#WARNING To speedup geom.AsText a bulk operator implementation is needed

# 14:02:52 >  
# 14:02:52 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-14721" "--port=35167"
# 14:02:52 >  

#SELECT ST_AsText(boundary) FROM named_places WHERE name = 'Goose Island';
% sys.%1 # table_name
% %1 # name
% clob # type
% 45 # length
[ "POLYGON ((67 13, 67 18, 59 18, 59 13, 67 13))"	]

# 14:02:52 >  
# 14:02:52 >  "Done."
# 14:02:52 >  

